### PR TITLE
New design

### PR DESCRIPTION
**Pull request by serhiy1017**
New design -> UAT

### CODE REVIEW

File: src/conf/config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 const app_config = {
 //	FYC_API_URL: process.env.FYC_API_URL,
+//	FYC_API_URL: "https://api-dev.findyourchurch.org",
 	FYC_API_URL: "https://api-uat.findyourchurch.org",
```
Comment (jjarl12) [inline src/conf/config.js:5]: So this is not being injected by environment variables. I shouldn't see the URL in here.

**Review by jjarl12 — CHANGES_REQUESTED** [review]
Please see comment and also I want only develop to merge into release-uat.

File: src/conf/config.js (hunk 1 above)
Comment (serhiy1017) [reply src/conf/config.js:5]: How about using a .env file? (FE can't access system environment variables.)
Our .env file can include:
`FYC_API_URL="https://api-uat.findyourchurch.org"`
To use this way, we must some node modules. I will notify you when I am absolutely sure about this.

Comment (jjarl12) [reply src/conf/config.js:5]: @serhiy1017 I'm sorry, but I thought I replied to this. Yes lets do a .env file.

**Comment by jjarl12** [comment]
Please have this all merged into develop. Only the develop branch should be merged into release-uat.

**Pull request closed without merging**